docs/guide/service/annotations.md

```diff
@@ -63,8 +63,8 @@ Traffic Routing can be controlled with following annotations:
 - <a name="lb-type">`service.beta.kubernetes.io/aws-load-balancer-type`</a> specifies the load balancer type. This controller reconciles those service resources with this annotation set to either `nlb-ip` or `external`.
 
     !!!note ""
-        - For `nlb-ip` type, controller will provision NLB with IP targets. This value is supported for backwards compatibility
-        - For `external` type, NLB target type depend on the annotation [nlb-target-type](#nlb-target-type)
+        - For `nlb-ip` type, controller will provision NLB with IP targets. This value is supported for backwards compatibility, but it is deprecated.
+        - For `external` type, NLB target type depend on the annotation [nlb-target-type](#nlb-target-type). If used in combination with [lb-scheme](#lb-scheme) set to internal, it will provision an internal NLB.
```

Comment:
Thanks for contributing this.
I think maybe ```- [Deprecated] `nlb-ip` type, controller will provision NLB with IP targets. This value is supported for backwards compatibility```

BTW, it feels a weird to mention  lb-scheme here as it have it's own section. and internal NLB is the default mode if lb-scheme is not specified after all.

Reply:
1st thing :+1: 

About the lb-scheme, I just wanted to highlight that "external" allows to use "internal" LBs. That's what happened to me, I did not try to use the external type because I thought that would only provision an internet-facing LB :sweat_smile:

Maybe if not a note in the lb-scheme clarifying that it can be used with the "external" type?

Reply:
@froblesmartin 
I understand your confusing now. it's a valid point. I think maybe add a separate tip section like 
```
!!!tip
   This annotation specifies the controller used to provision LoadBalancers. Refer to [lb-scheme](#lb-scheme) to specify whether the LoadBalancer is public-facing.
```

Reply:
btw, this annotation itself will be deprecated soon as well, we'll use new spec.ServiceClass in the future instead of `service.beta.kubernetes.io/aws-load-balancer-type`

Reply:
Good to know! :smile: I have changed it to the tip, and as well reduced the last part just to clarify that this controller can provision any NLB+Target Group configuration.